android/build.gradle

```
@@ -26,6 +26,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
+    namespace 'com.linecorp.flutter_line_sdk'
```

Could we check `hasProperty` before setting namespace?

```suggestion
    if (project.android.hasProperty('namespace')) {
        namespace 'com.linecorp.flutter_line_sdk'
    }
```